splunk-otel-android/src/main/java/com/splunk/rum/SpanFilterBuilder.java

```diff
@@ -37,6 +38,19 @@
     SpanFilterBuilder() {
     }
 
+    /**
+     * Remove matching spans from the exporter pipeline.
+     * <p>
+     * Spans with names that match the passed {@code pattern} will not be exported.
+     *
+     * @param pattern A regular expression pattern that matches names of spans that should be
+     *                rejected.
+     * @return {@code this}.
+     */
+    public SpanFilterBuilder rejectSpansByName(Pattern pattern) {
```

Comment:
I think we should hold off on adding this method until we get feedback that it's definitely necessary. It's so easy for the end user to implement this themselves using the `rejectSpansByName(Predicate<String>)` method that I think we can start that just that one.

Reply:
Got it 👍 
Let's close it for now and wait for more feedback on span filtering then.